src/CommunityToolkit.Maui.MediaElement/Options/MediaOptions.macios.cs

```diff
@@ -0,0 +1,26 @@
+using AVFoundation;
+
+namespace CommunityToolkit.Maui;
+
+partial class MediaOptions
```

Comment:
I would say those `Options` should be immutable. So we can refactor it to be a record and remove the `setter`s